sources/packages/web/src/views/aest/institution/ApproveDenyDesignation.vue

```diff
@@ -1,33 +1,75 @@
 <template>
-  <!-- CSS class temporary-modal is work around for lack of responsiveness of v-dialog. -->
-  <modal-dialog-base
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-    :title="title"
-  >
-    <template v-slot:content>
-      <div class="temporary-modal">
-        <formio
-          formName="approvedenydesignation"
-          :data="designation"
-          @loaded="formLoaded"
-          @submitted="submitDesignationUpdate"
-        ></formio>
-      </div>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="Role.InstitutionApproveDeclineDesignation">
-        <template #="{ notAllowed }">
+  <v-form ref="approveDenyDesignation">
+    <modal-dialog-base :showDialog="showDialog" :title="title" max-width="730">
+      <template #content>
+        <error-summary :errors="approveDenyDesignation.errors" />
+        <span class="label-value">{{ subTitle }} </span>
+        <content-group class="my-3" v-if="showApprovalContent">
+          <span class="label-bold color-blue">Effective dates</span>
+          <v-row
+            ><v-col
+              ><v-text-field
+                label="Start date"
+                class="mt-2"
+                type="date"
+                placeholder="yyyy-MM-dd"
+                v-model="formModel.startDate"
+                variant="outlined"
+                :rules="[checkStringDateFormatRule]"
```

Comment:
We have been using `hide-details="auto"` to remove the extra space below the inputs, right?